UX-Guide-Metadata/draft/principles/index.html

```diff
@@ -1101,17 +1101,17 @@ <h3 data-localization-id="legal-considerations-title">Legal considerations</h3>
 					is to provide as much accessibility information as is possible.</p>
 
 				<section id="legal-examples">
-					<h4>Examples</h4>
-					<p>TBD</p>
-					<!--
+					<h4>Example</h4>
+
+					
+<p>In many cases no legal exception or exemption information will be presented. Where it it is provided, the example is  provided as   a possible descriptive   explanation.</p>
```

Comment:
I think this could read more simply as: "The following example provides as a possible descriptive explanation when legal exception or exemption information needs to be presented."

At a minimum, there is an "it it" that needs fixing in the original.

```suggestion
<p>The following example provides as a possible descriptive explanation when legal exception or exemption information needs to be presented.</p>
```